ChangeLog.md

```diff
@@ -179,6 +179,8 @@
 
   * Fix Python linter errors in (Matt West).
 
+  * Fix flaky test cases in `pl-number-input` which didn't handle comparison options properly (Mingjie Zhao).
```

Comment:
The changelog is ordered chronologically, so this should be at the bottom of the "Fix" section.

Reply:
Hmmm I thought I was ordered lexigraphically...

Reply:
Thanks!